test/powershell/Language/Parser/Parser.Tests.ps1

```diff
@@ -518,6 +518,8 @@ Describe "ParserTests (admin\monad\tests\monad\src\engine\core\ParserTests.cs)"
 
 	It "Test that functions are resolved before cmdlets. (line 1678)"{
         $result = ExecuteCommand 'function testcmd-parserBVT { 3 };testcmd-parserBVT'
+        $PowerShell.AddScript(". $functionDefinitionFile").Invoke()

```

Comment:
hm, that's not good. Can we just use something else, not `testcmd-parserBVT` in the above line?